app/settings/page.tsx

```diff
@@ -0,0 +1,225 @@
+'use client'
+import React, { useState } from 'react';
+import { Camera, Lock, Bell, Globe, Shield, LogOut, Leaf } from 'lucide-react';
+// import { Alert, AlertDescription } from '@/components/ui/alert';
+
+const ProfileSettings = () => {
+  const [saved, setSaved] = useState(false);
+  const [selectedInterests, setSelectedInterests] = useState([
+    'community-and-advocacy',
+    'green-transportation'
+  ]);
+
+
+  const sustainabilityInterests = [
+    { id: 'eco-friendly-living', emoji: '🌱', label: 'Eco-Friendly Living' },
+    { id: 'recycling-upcycling', emoji: '♻️', label: 'Recycling & Upcycling' },
+    { id: 'climate-action', emoji: '🌍', label: 'Climate Action' },
+    { id: 'green-transportation', emoji: '🚴', label: 'Green Transportation' },
+    { id: 'reforestation', emoji: '🌳', label: 'Reforestation' },
+    { id: 'renewable-energy', emoji: '🌞', label: 'Renewable Energy' },
+    { id: 'sustainable-fashion', emoji: '🛍️', label: 'Sustainable Fashion' },
+    { id: 'plastic-free-living', emoji: '🍃', label: 'Plastic-Free Living' },
+    { id: 'community-and-advocacy', emoji:'👫', label:'Community and Advocacy'},
+    { id: 'sustainable-development-goals-(SDGs)', emoji:'🌟', label:'Sustainable Development Goals (SDGs)'},
+  ];
+  
+  const handleSave = () => {
+    setSaved(true);
+    setTimeout(() => setSaved(false), 3000);
+  };
+
+  const toggleInterest = (interestId:string) => {
+    setSelectedInterests(prev => 
+      prev.includes(interestId)
+        ? prev.filter(id => id !== interestId)
+        : [...prev, interestId]
+    );
+  };
+
+
+  return (
+    <div className="min-h-screen bg-white">
+      {/* Header */}
+      <div className="bg-green-50 p-6 border-b border-green-100">
+        <h1 className="text-2xl font-semibold text-gray-800">Profile Settings</h1>
+        <p className="text-gray-600">Manage your EcoSync account settings and preferences</p>
+      </div>
+
+      <div className="max-w-4xl mx-auto p-6">
+        {/* {saved && (
+        //   <Alert className="mb-6 bg-green-50 border-green-200">
+        //     <AlertDescription>Settings saved successfully!</AlertDescription>
+        //   </Alert>
+        )} */}
+
+        {/* Profile Section */}
+        <div className="mb-8">
+          <div className="flex items-center gap-6">
+            <div className="relative">
+              <img 
```

Comment:
Use <Image/>  from next js not the normal <img /> tag won't work in nextjs ckeck the UI .